src/celeritas/global/CoreParams.cc

```diff
@@ -156,6 +157,12 @@ CoreParams::CoreParams(Input input) : input_(std::move(input))
     input_.action_reg->insert(std::make_shared<InitializeTracksAction>(
         input_.action_reg->next_id()));
 
+    // Construct sort tracks action
+    input_.action_reg->insert(std::make_shared<SortTracksAction>(
+        input_.action_reg->next_id(),
+        ActionOrder::start,
```

Comment:
In `ActionSequence` we sort the actions by `action_order` and increasing action ID, so this *should* definitely come after the initialization action, which can set "inactive" slots to "alive". So we should either make a note here that the action needs to be created *after* `InitializeTracksAction`, or (probably better) make a new `sort_start` `ActionOrder` that guarantees the correct placement.

Reply:
Good point, adding a new `ActionOrder` variant makes it less likely to break.